rcp/org.testeditor.rcp4.views.tcltestrun/src/org/testeditor/rcp4/views/tcltestrun/TclLauncherUi.xtend

```diff
@@ -74,6 +84,24 @@ class TclLauncherUi implements Launcher {
 		return false
 	}
 
+	def createTestCaseIndex(IProject project) {
```

Comment:
I think we don't need this method at all. When we need a resource to a given URI we can use `IResourceDescriptions.getResourceDescription(URI)`. No need to fill a custom map beforehand.